domains/client/domain-operator/src/tests.rs

```diff
@@ -8800,3 +8800,137 @@ fn proof_to_tx(
         .construct_unsigned_extrinsic(pallet_domains::Call::submit_fraud_proof { fraud_proof })
         .into()
 }
+
+// E2E test: call transporter precompile (0x0800) via EVM tx and observe XDM processing
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
This test should work with any number of threads, in particular, it needs to work with the default number of tokio threads. (Because that's what we use in production.)

```suggestion
#[tokio::test(flavor = "multi_thread")]
```